web/src/beta/features/Editor/tabs/story/SidePanel/ContentStory/index.tsx

```diff
@@ -53,7 +53,7 @@ const ContentStory: React.FC<Props> = ({
           }}
           renderItem={item => {
             return (
-              <PageItemWrapper pageCount={item.index + 1} isSwipable={item.index % 2 === 0}>
+              <PageItemWrapper pageCount={item.index + 1} isSwipeable={item.index % 2 === 0}>
```

Comment:
We can just do `item.index === 0` here, since triple equals is strict.
As reference:
```
undefined == 0 // true
undefined === 0 // false
```